lib/api/core/plugins/pluginsManager.js

```diff
@@ -219,7 +237,7 @@ class PluginsManager {
    * @returns {string[]}
    */
   getStrategyFields (strategyName) {
-    return this.strategies[strategyName].strategy.config.fields;
+    return this.strategies[strategyName].strategy.config.fields || [];
```

Comment:
I suggest to prevent the TypeError by checking the existence of `this.strategies[strategyName]` before returning anything.